### ISSUE TITLE
Avatar img alt text clashing with validator name

### ISSUE THREAD

**Issue by sam-keen** [vegaprotocol/frontend-monorepo#898]
If a validator lacks an avatar url, the alt text clashes:

![Screen Shot 2022-07-28 at 11 56 29 AM](https://user-images.githubusercontent.com/2410498/181494888-1b2c6763-54c4-4d0d-bd77-60b183db5594.jpg)
